music21/pitch.py

```diff
@@ -4511,6 +4561,16 @@ def transpose(
         >>> pc6.transpose(10, inPlace=True)
         >>> pc6.spellingIsInferred
         True
+
+        Test an issue with inPlace not setting microtone.
+
+        >>> dFlatAndAHalf = pitch.Pitch('D2-`')
```

Comment:
can you rewrite without the back-tick but instead explicitly set a 3qtr flat accidental object?  I'm planning on removing the backtick and ~ abbreviations -- probably a mistake to include them originally.

Reply:
Done.